src/Controls/src/Core/Handlers/Items2/iOS/ItemsViewController2.cs

```diff
@@ -410,7 +410,7 @@ internal void UpdateView(object view, DataTemplate viewTemplate, ref UIView uiVi
 
 				uiView?.Dispose();
 				uiView = null;
-
+				formsElement?.Handler?.DisconnectHandler();
```

Comment:
Do we need the dispose above? doesn't the disconnect do it? And also, @PureWeen said dispose is the evil path to crashes.

Reply:
We should remove the `Dispose`

We never want to call `Dispose` on anything. 
We should also make sure all the code that's inside any `Dispose` methods related to CV get moved to a disconnect path

Reply:
https://github.com/dotnet/maui/issues/26134

Reply:
We can try move the Dipose calls all in another PR for #26134